### PR TITLE
Development publish UI package

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
## Description

This pull request includes updates to the PDF Signer and Script Editor documentation and example files. The changes align the documentation with recent updates in the repository.

## Changes

- Updated the PDF Signer Web Component documentation to reflect recent changes in the entry point, package structure, and usage instructions.
- Similarly, updated the Script Editor Web Component documentation to match the recent changes in the entry point, package structure, and usage instructions.
- Adjusted the example HTML code for both components to ensure clarity and consistency.
- Updated the example server and configuration files for better integration with the PDF Signer component.

## NPM Package Ready

The changes in this pull request make the package ready to be published on npm. All documentation is updated, and the examples are aligned with the latest package structure and usage instructions.

## Related issues
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1542

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by 0oM4R — CHANGES_REQUESTED** [review]
v server have syntax errors

File: packages/UI/.env [hunk 1]
```diff
@@ -1 +1 @@
-VITE_MNEMONIC="<your_mnemonic_here>"
\ No newline at end of file
+# VITE_MNEMONIC="<your_mnemonic_here>"
```
Comment (0oM4R) [inline packages/UI/.env:1]: I see this env var is used in component, why should we comment it ?

Comment (Mahmoud-Emad) [reply packages/UI/.env:1]: As outlined in the documentation, there are two provider options available:

1. **Environment Variable**: Users can pass the `mnemonic` as an environment variable, and the account will be loaded accordingly.

2. **Threefold Connector Extension**: Alternatively, users can comment the line `VITE_MNEMONIC="<your_mnemonic_here>"` in the `.env` file, following the documentation instructions. This step is essential for utilizing the `Threefold Connector Extension`.

It's crucial to note that, when we publish the package on npm, users won't be able to directly pass their mnemonic. Therefore, the recommended approach is to comment out the `VITE_MNEMONIC` variable in the `.env` file, effectively unlocking the functionality provided by the `Threefold Connector Extension` for users.

**Review by 0oM4R — APPROVED** [review]
- Pdf viewer works fine :
Screenshot from 2023-11-29 16-09-35
- script editor also:
Screenshot from 2023-11-29 16-09-35

- style looks good here :
image

both of them on node server

**Review by amiraabouhadid — APPROVED** [review]
looks good !

Screenshot from 2023-11-29 16-38-13

**Pull request merged**